src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -2877,33 +2877,27 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
 
         case NI_Vector128_Sum:
         case NI_Vector256_Sum:
+        case NI_Vector512_Sum:
         {
             assert(sig->numArgs == 1);
             var_types simdType = getSIMDTypeForSize(simdSize);
```

Comment:
@DeepakRajendrakumaran don't you need a ISA check here for simdSize = 64 ?

Reply:
@EgorBo I fixed the library test failures. But see a superpmi fail. Looks like network issue? - https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-95568-merge-68d27cc0ef224aaaa9/osx-arm64-1/1/console.081ce495.log?helixlogtype=result

Reply:
I've restarted the jobs

Reply:
Thanks. It passes now!